gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoints-group/api-endpoints-group.component.html

```diff
@@ -0,0 +1,51 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="back-button">
+  <gio-go-back-button [ajsGo]="{ to: 'management.apis.ng.endpoints' }"></gio-go-back-button>
```

Comment:
Here I'm using the standard GIO back button but on the endpoint V4 page this is used
```
<div class="back-button">
  <a mat-icon-button class="mat-body-strong" [uiSref]="'management.apis.ng.endpoints'"
    ><mat-icon svgIcon="gio:arrow-left"></mat-icon> Go back to your endpoints</a
  >
</div>
```
What is preferred ? I would think it is the standard GIO back button.